### PR TITLE
Add the siblings table view

### PR DESCRIPTION
**Pull request by mprahl**
This also has some somewhat unrelated fixes that made this easier to develop. Those are separated by commit.

### CODE REVIEW

**Comment by mprahl** [comment]
@sarah256 feel free to review this as well.

File: src/app/pipes/propertydisplay.ts [hunk 1]
```diff
@@ -20,3 +20,24 @@ export class PropertyDisplayPipe implements PipeTransform {
     return rv;
   }
 }
+
+@Pipe({name: 'propertyValueDisplay'})
```
Comment (yashvardhannanavati) [inline src/app/pipes/propertydisplay.ts]: There is a function doing the same thing in `storysidebar.component.ts` . Is it possible to replace that function with this pipe too?

Comment (mprahl) [reply src/app/pipes/propertydisplay.ts]: I believe I did. Can you not find it?

Comment (yashvardhannanavati) [reply src/app/pipes/propertydisplay.ts]: ah, found it! sorry about that, i had to dig it up :p

**Comment by sarah256** [comment]
:+1:

**Comment by yashvardhannanavati** [comment]
:+1:

**Pull request merged**